bigtable/api/Snippets/Writes/WriteBatch.cs

```diff
@@ -37,7 +37,7 @@ public class WriteBatch
 
             TableName tableName = new TableName(projectId, instanceId, tableId);
             BigtableVersion timestamp = new BigtableVersion(DateTime.UtcNow);
-            String COLUMN_FAMILY = "stats_summary";
+            string COLUMN_FAMILY = "stats_summary";
```

Comment:
As a follow-up we should probably rename this and make it a const (even if it's still as a local variable)

Reply:
Yes, I think we shoulc rewrite a lot of these anyway. Let's what we can get done medium term.